geoips/plugins/modules/procflows/single_source.py

```diff
@@ -605,6 +605,8 @@ def apply_interp_after_alg(
     # If required, interpolate the result prior to returning
     elif prod_plugin.family == "algorithm_interpolator_colormapper":
         interp_args["varlist"] = [prod_plugin.name]
+        if prod_plugin.name == "octopy":
```

Comment:
Yes this needs to be. This was just a workaround I implemented for the time being. Kept running into errors where u, v weren't being interpolated but I needed them to be.

Reply:
Could you please create an issue to generalize this, link to this conversation from the issue, link to the issue from this conversation, then you can resolve this conversation?

Reply:
Before I create an issue on this, there might be a workaround. Do you know how to get multiple variables interpolated in GeoIPS? I basically have three outputs of my algorithm which stem from two abi scan times of ``B13Rad``. I generate variables ``["octopy", "u", "v"]`` from my algorithm. ``u`` and ``v`` would be nice to have because I'm able to add quivers to the image if I want to. So, I'm guessing it'd be similar to the fashion in which you produce your windbarb imagery.

Reply:
Ok, I just pushed an update that should generalize this. If you're using the family ``algorithm_interpolator_colormapper``, we assume that we will only be interpolating a variable that matches your product's name. However I think that's a shortsighted assumption as a user may produce other variables from their algorithm, on top of their product name. For example, in octopy, I produce a variable named ``octopy``, as well as``u, v`` vectors which can be used to add quivers to the output. If you look at the updated code of this section, this should be fixed now.

Reply:
I think I'm still going to create an issue about this point, but in a different light. After @jsolbrig and I spoke, we thought it may be a good idea for interpolators to be agnostic of the incoming data. Just interpolate what's been passed to it, regardless of name. Would probably clean up some logic downstream as well. Only thing we'd need to consider after that is how output formatters will handle incoming data.

Reply:
See #819.